openapi/action.yml

```
@@ -13,34 +12,84 @@ inputs:
     description: "Input file to use to generate openAPI docs"
     required: true
     default: ""
+  validate:
+    description: "Validate OpenAPI spec before generating docs"
+    required: false
+    default: "true"
+  redocly-version:
+    description: "Version of Redocly CLI to use"
+    required: false
+    default: "latest"
+  config:
+    description: "Path to the Redocly config file for linting"
+    required: false
 
 runs:
   using: "composite"
   steps:
-    ## checkout the code
     - name: Checkout the latest code
       id: git_checkout
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
-    ## Use redoc to generate HTML of openapi.yml
-    - name: Redoc
-      id: run_redoc
-      uses: seeebiii/redoc-cli-github-action@c9649b33918da5eb290b81cd03a943caea091540 # v10
+    - name: Setup Node.js
+      uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
       with:
-        args: "bundle -o ${{ inputs.output }} ${{ inputs.input }}"
+        node-version: "20"
+
+    - name: Install Redocly CLI
+      id: install_redocly
+      shell: bash
+      run: |
+        if [ "${{ inputs.redocly-version }}" = "latest" ]; then
+          npm install -g @redocly/cli
+        else
+          npm install -g @redocly/cli@${{ inputs.redocly-version }}
```

could this simply be:
```suggestion
        npm install -g @redocly/cli@${{ inputs.redocly-version }}
```